samples/ContainmentSuperVsExtendsSameType.java

```diff
@@ -19,8 +19,10 @@
 @NullMarked
 class ContainmentSuperVsExtendsSameType {
   void x() {
+    // `conflicting_annotations` b/c upper and lower bound of `? super Object` differ.
     // :: error: jspecify_nullness_mismatch jspecify_but_expect_nothing
-    new Check<Lib<? extends Number>, Lib<? super Number>>();
+    // :: error: jspecify_conflicting_annotations
+    new Check<Lib<? extends Object>, Lib<? super Object>>();
```

Comment:
#485 previously changed this, but that change introduced a java compilation failure that went unnoticed.